fdbserver/ClusterController.actor.cpp

```diff
@@ -3,7 +3,7 @@
  *
  * This source file is part of the FoundationDB open source project
  *
- * Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+ * Copyright 2013-2019 Apple Inc. and the FoundationDB project authors
```

Comment:
Do we need to change this every year for all of files?

Reply:
This is a more general ask: how do we keep the copyright headers up to date in all FDB open sourced projects.

Reply:
Maybe Copyright since 2013?

Reply:
Not sure how other people is doing. I am just changing files that I am working on.